Makefile

```diff
@@ -10,7 +10,7 @@ all: sitegen site
 
 sitegen: bin
 	cd sitegen && dotnet build
-	cp sitegen/SiteGenerator.ConsoleApp/bin/Debug/netcoreapp3.1/* bin/
+	cp -r sitegen/SiteGenerator.ConsoleApp/bin/Debug/net5.0/* bin/
```

Comment:
This is required because of https://github.com/perlun/sitegen/pull/50. The output binaries are now in a differen path after building sitegen.